R/etwfe.R

```diff
@@ -103,7 +105,12 @@ etwfe = function(
   if (is.numeric(gvar)) gvar = names(data)[gvar]
   ivar = eval(substitute(ivar), nl, parent.frame())
   if (is.numeric(ivar)) ivar = names(data)[ivar]
+  intvar = eval(substitute(intvar), nl, parent.frame())
+  if (is.numeric(intvar)) intvar = names(data)[intvar]
+  id = eval(substitute(id), nl, parent.frame())
+  if (is.numeric(id)) id = names(data)[id]
   
+  if (!is.null(intvar) & is.null(id)) stop("The individual-ID is required with an interaction term.\n")
```

Comment:
I might be misunderstanding what's happening here, but is there a reason that the id (xref) cannot be guessed at from the data, like we do for `gref` and `tref`?